### PR TITLE
Re-write of get_address_table()

### PR DESCRIPTION
**Pull request by PseudotsugaMeziesii**
TLDR; Used the header lines from command output to determine location of relevant information in subsequent rows.

Sorry if it's a bit presumptuous, but I decided to try and tackle a re-write of the get_mac_address() function. There was a lot of reliance on a custom regex for every single edge case that came up. It was starting to get pretty tough to follow. Cisco certainly didn't make it easy for us...

For the proposed changes, an attempt was made to leverage the header line of the command output to provide a reference for the subsequent rows containing MAC addresses (or interfaces with commas). There are still some kooky workarounds, but hopefully there might be fewer custom considerations required in the future.

Please take a look at what was changed and hit me up with any concerns. All testing was completed using the existing test files, and I tried to follow current conventions.

### CODE REVIEW

**Comment by PseudotsugaMeziesii** [comment]
NM. I'm a douche. Failed testing...
